tests/issue-531.py

```diff
@@ -0,0 +1,14 @@
+import sys
+
+def cli_update_repo():

```

Comment:
This shouldn't be a test, it needs to be integrated into the cli, so `taf\tools\repo\__init__.py`

Reply:
I edited the code to import click according to __init__.py file.